### ISSUE TITLE
Bump to clean v0.24 when available

### ISSUE THREAD

**Issue by davefrey** [liquidvotingio/decidim-module-liquidvoting#68]
This issue is to bump our module to Decidim v0.24.0

See also the bump for our Demo app liquidvotingio/liquid-decidim-demo#15

Checklist:
- [x] Read Decidim [Release v0.24.0 · decidim/decidim · GitHub](https://github.com/decidim/decidim/releases/tag/v0.24.0)
  - [x] Note possible regression/conflict items
  - [ ] Raise to team Decidim evolutions we might want to pursue/integrate
  - [x] Document concerns/resolutions in the checklist issue, to help next bump effort
- [x] Make a new bump branch from clean local workspace on master
- [x] Change `DECIDIM_VERSION` and `VERSION` in the `version.rb` file, and update `Gemfile.lock`:
```
bundle install
```
- [x] Regenerate the test app, to get latest Decidim migrations and a new (test) database. Note we don't add any migrations to this module via `rake decidim:update`, that happens in our demo app.
```
bundle exec rake test_app
```
- [x] Inspect Decidim code changes since our last bump [Comparing v0.24.0.rc2…v0.24.0 · decidim/decidim · GitHub](https://github.com/decidim/decidim/compare/v0.24.0.rc2...v0.24.0)
  - [x] Especially but not only files in the `decidim-proposals` module; propose keyword searching terms of interest
  - [x] Incorporate Decidim evolutions, and try to minimize the diff noise between latest Decidim code and our overrides
  - [x] Especially but not only check `app/overrides/` files (maybe specs have evolved in a way we want to incorporate)
- [x] For any issues:
  - [x] Fix the blockers on the bump branch
  - [x] Log issues for things that can wait
- [ ] Any updates to our `README.md`?
- [x] Keep specs and rubocop green
```
bundle exec rspec
bundle exec rubocop
```

**Comment by oliverbarnes** [comment]
v0.24 is out. Changelog: https://github.com/decidim/decidim/releases/tag/v0.24.0

**Comment by oliverbarnes** [comment]
Blog post about it: https://decidim.org/blog/en/2021-03-26-new-version-0-24-0/

**Comment by davefrey** [comment]
I'll pause this since we really want the version with the mimemagic fix, maybe `v0.24.1` or `v0.25.0`.

See also #73

**Comment by oliverbarnes** [comment]
Seems like so far Decidim fixes this for them with mimemagic 0.3.6: https://github.com/decidim/decidim/pull/7701 

And that's included in the v0.24.0 release: see last item in https://github.com/decidim/decidim/releases/tag/v0.24.0

**Comment by davefrey** [comment]
Updated issue description to be a bump checklist that we can copy/edit/improve on future bumps.
